projects/gui/src/gamedatabasedlg.cpp

```diff
@@ -527,11 +544,39 @@ void GameDatabaseDialog::createOpeningBook()
 	task->start();
 }
 
+void GameDatabaseDialog::copyGame()
+{
+	QString str("");
```

Comment:
I think empty strings are usually initialized with the default constructor, so `QString str;` should be enough.

Reply:
Ah, thank you. Have been unsure at that moment.